packages/components/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -5,11 +5,17 @@ import React, { FC, Fragment } from 'react'
 import { Checkbox } from '../Checkbox/Checkbox'
 import { ComponentWrapper } from '../../wrapper'
 import { Error, Header, Hint } from '../shared/FormText'
-import { FormElementProps, TextWithA11y } from '../../types'
+import {
+  FormElementProps,
+  StringOrTextWithA11y,
+  TextWithA11y,
+} from '../../types'
 import { Spacer } from '../Spacer/Spacer'
 import { useTheme } from '../../utils'
 
 type TextWithA11yAndValue = TextWithA11y & {
```

Comment:
Name getting somewhat dated with addition of `testID` and now `description`, perhaps rename to `ItemsProps` or something.